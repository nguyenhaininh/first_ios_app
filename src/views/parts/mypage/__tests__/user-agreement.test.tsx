/*---------------------------------------------------------------------------------------------
 *  Copyright (c) Lunascape Corporation. All rights reserved.
 *--------------------------------------------------------------------------------------------*/
import * as React from 'react';
import { createMemoryHistory } from 'history';
import UserAgreement from '../user-agreement';

import * as renderer from 'react-test-renderer';

it('renders correctly', () => {
  const history = createMemoryHistory();
  const tree = renderer.create(
    <UserAgreement history={history} />
  ).toJSON();
  expect(tree).toMatchSnapshot();
});
