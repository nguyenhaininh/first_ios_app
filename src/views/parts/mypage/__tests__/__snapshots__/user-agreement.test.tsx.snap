// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  style={
    Object {
      "backgroundColor": "#f7f7f7",
      "flex": 1,
      "paddingTop": 20,
    }
  }
>
  <View
    accessible={true}
    isTVSelectable={true}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "opacity": 1,
      }
    }
  >
    <View
      style={
        Array [
          Object {
            "alignItems": "center",
            "backgroundColor": "#ffffff",
            "borderBottomColor": "#eaebed",
            "borderBottomWidth": 0.5,
            "flexDirection": "row",
            "height": 54,
            "paddingHorizontal": 15,
          },
          Object {
            "borderTopColor": "#eaebed",
            "borderTopWidth": 0.5,
          },
        ]
      }
    >
      <Text
        accessible={true}
        allowFontScaling={true}
        ellipsizeMode="tail"
        style={
          Object {
            "color": "#333333",
            "flex": 1,
            "fontSize": 16,
          }
        }
      >
        T会員規約
      </Text>
      <Image
        source={
          Object {
            "testUri": "../../../src/assets/images/arrow-ico.png",
          }
        }
      />
    </View>
  </View>
  <View
    accessible={true}
    isTVSelectable={true}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "opacity": 1,
      }
    }
  >
    <View
      style={
        Array [
          Object {
            "alignItems": "center",
            "backgroundColor": "#ffffff",
            "borderBottomColor": "#eaebed",
            "borderBottomWidth": 0.5,
            "flexDirection": "row",
            "height": 54,
            "paddingHorizontal": 15,
          },
          null,
        ]
      }
    >
      <Text
        accessible={true}
        allowFontScaling={true}
        ellipsizeMode="tail"
        style={
          Object {
            "color": "#333333",
            "flex": 1,
            "fontSize": 16,
          }
        }
      >
        Tマガジン利用規約
      </Text>
      <Image
        source={
          Object {
            "testUri": "../../../src/assets/images/arrow-ico.png",
          }
        }
      />
    </View>
  </View>
  <View
    accessible={true}
    isTVSelectable={true}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "opacity": 1,
      }
    }
  >
    <View
      style={
        Array [
          Object {
            "alignItems": "center",
            "backgroundColor": "#ffffff",
            "borderBottomColor": "#eaebed",
            "borderBottomWidth": 0.5,
            "flexDirection": "row",
            "height": 54,
            "paddingHorizontal": 15,
          },
          null,
        ]
      }
    >
      <Text
        accessible={true}
        allowFontScaling={true}
        ellipsizeMode="tail"
        style={
          Object {
            "color": "#333333",
            "flex": 1,
            "fontSize": 16,
          }
        }
      >
        TSUTAYA決済サービス利用規約
      </Text>
      <Image
        source={
          Object {
            "testUri": "../../../src/assets/images/arrow-ico.png",
          }
        }
      />
    </View>
  </View>
</View>
`;
